examples/offline_inference_distributed_cont_batching.py

```diff
@@ -0,0 +1,78 @@
+"""
```

Comment:
Wondering why the script is called `*_cont_batching`. Why is continuous batching specifically mentioned here?